lib/child_process.js

```diff
@@ -88,6 +88,13 @@ const {
 
 const MAX_BUFFER = 1024 * 1024;
 
+/**
+ * Spawns a new Node.js insatnce
+ * using the execPath of the parent process
+ * @param {string} modulePath
```

Comment:
It seems this is lacking a definition for the optional parameters (`args` and `options`).